From 25ca1b05529d7ce95a3948f45391e22ab0fd9744 Mon Sep 17 00:00:00 2001
From: Jakub Filak <jfilak@redhat.com>
Date: Wed, 12 Aug 2015 17:40:12 +0200
Subject: [PATCH] ccpp: use global TID

'%i' is TID of the thread from the crashed process's PID namespace but
abrt-hook-ccpp is run by kernel in the init PID namespace, so '%i'
is TID of a completely unrelated process.

For mode details see 'man 5 core'.

Related: rhbz#1252590

Signed-off-by: Jakub Filak <jfilak@redhat.com>
---
 src/hooks/abrt-install-ccpp-hook.in | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/hooks/abrt-install-ccpp-hook.in b/src/hooks/abrt-install-ccpp-hook.in
index fff0a33..65f771f 100755
--- a/src/hooks/abrt-install-ccpp-hook.in
+++ b/src/hooks/abrt-install-ccpp-hook.in
@@ -11,9 +11,9 @@ SAVED_PATTERN_DIR="@VAR_RUN@/abrt"
 SAVED_PATTERN_FILE="@VAR_RUN@/abrt/saved_core_pattern"
 HOOK_BIN="@libexecdir@/abrt-hook-ccpp"
 # Must match percent_specifiers[] order in abrt-hook-ccpp.c:
-PATTERN="|$HOOK_BIN %s %c %p %u %g %t %e %i"
+PATTERN="|$HOOK_BIN %s %c %p %u %g %t %e %I"
 # Same, but with bogus "executable name" parameter
-PATTERN1="|$HOOK_BIN %s %c %p %u %g %t e %i"
+PATTERN1="|$HOOK_BIN %s %c %p %u %g %t e %I"
 
 # core_pipe_limit specifies how many dump_helpers can run at the same time
 # 0 - means unlimited, but it's not guaranteed that /proc/<pid> of crashing
-- 
2.4.3

